frame/democracy/src/benchmarking.rs

```diff
@@ -29,21 +29,22 @@ use frame_system::{Pallet as System, RawOrigin};
 use sp_runtime::traits::{BadOrigin, One};
 
 use crate::Pallet as Democracy;
-
-const DOT_ASSET: u64 = 2;
+use sp_std::vec;
 
 const SEED: u32 = 0;
 const MAX_REFERENDUMS: u32 = 99;
 const MAX_SECONDERS: u32 = 100;
 const MAX_BYTES: u32 = 16_384;
 
+const DOT_ASSET: u128 = 2;
+
 fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
 fn funded_account<T: Config>(name: &'static str, index: u32) -> T::AccountId {
 	let caller: T::AccountId = account(name, index, SEED);
-	//T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
+	// T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
 	caller
 }
```

Comment:
dus the `account` function add funding?

Reply:
not clear about this ?

Reply:
ill just remove it , it was part of the original tests that came from the substrate repo, they commented it out that way

Reply:
I mean part of the previous democracy , ill update and push

Reply:
I just pushed an update , removed funded_account and called account directly

Reply:
l'll update this and